Pod/Classes/Core/AREmission.m

```diff
@@ -54,57 +65,29 @@ + (instancetype)sharedInstance;
   return _sharedInstance;
 }
 
-- (instancetype)initWithUserID:(NSString *)userID
-           authenticationToken:(NSString *)authenticationToken;
-{
-  return [self initWithUserID:userID
-          authenticationToken:authenticationToken
-                  packagerURL:nil
-        useStagingEnvironment:NO];
-}
-
-- (instancetype)initWithUserID:(NSString *)userID
-           authenticationToken:(NSString *)authenticationToken
-                   packagerURL:(nullable NSURL *)packagerURL
-         useStagingEnvironment:(BOOL)useStagingEnvironment;
-{
-  return [self initWithUserID:userID
-          authenticationToken:authenticationToken
-                  packagerURL:packagerURL
-        useStagingEnvironment:useStagingEnvironment
-                    sentryDSN:nil];
-}
-
-- (instancetype)initWithUserID:(NSString *)userID
-           authenticationToken:(NSString *)authenticationToken
-                   packagerURL:(nullable NSURL *)packagerURL
-         useStagingEnvironment:(BOOL)useStagingEnvironment
-                     sentryDSN:(nullable NSString *)sentryDSN;
+- (instancetype)initWithConfiguration:(AREmissionConfiguration *)config packagerURL:(nullable NSURL *)packagerURL
 {
-  NSParameterAssert(userID);
-  NSParameterAssert(authenticationToken);
+  NSParameterAssert(config);
```

Comment:
The config object should probably also still have some asserts for the individually properties.